docs/features.md

```diff
@@ -90,6 +90,24 @@ func (_m *Handler) HandleMessage(m pubsub.Message) error {
 }
 ```
 
+Generic type constraints can also be replaced by targeting the changed parameter with the square bracket notation on the left hand side.
+
+```shell
+mockery --replace-type github.com/vektra/mockery/v2/baz/internal/foo.InternalBaz[T]=baz:github.com/vektra/mockery/v2/baz.Baz
+```
+
+If a type constraint needs to be removed and replaced with a type, target the constraint with square brackets and include a '-' in front to have it removed.
+
+```shell
+mockery --replace-type github.com/vektra/mockery/v2/baz/internal/foo.InternalBaz[-T]=baz:github.com/vektra/mockery/v2/baz.Baz
+```
+
+When replacing a generic constraint, you can replace the type with a pointer by adding a '*' before the output type name.
+
+```shell
+mockery --replace-type github.com/vektra/mockery/v2/baz/internal/foo.InternalBaz[-T]=baz:github.com/vektra/mockery/v2/baz.*Baz
+```
+
```

Comment:
I think it would also be helpful to show code examples of what this would translate to.